extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/ResteasyReactiveJacksonProcessor.java

```diff
@@ -94,11 +94,6 @@ ServerDefaultProducesHandlerBuildItem jsonDefault() {
         return ServerDefaultProducesHandlerBuildItem.json();
     }
 
-    @BuildStep
-    ResteasyReactiveJacksonProviderDefinedBuildItem jacksonRegistered() {
-        return new ResteasyReactiveJacksonProviderDefinedBuildItem();
-    }
-
```

Comment:
Let's keep the build item and continue to produce for now. Make it deprecated and we will remove it in a few versions.

(There might be external extensions relying on it)